.github/workflows/eslint.yml

```diff
@@ -0,0 +1,13 @@
+name: CI
+on: [ push, pull_request ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install modules
+      run: yarn install --non-interactive --ignore-optional
+
+    - name: Run ESLint
+      run: node_modules/eslint/bin/eslint.js . --report-unused-disable-directives
```

Comment:
Ho about `yarn eslint`? This should execute local Eslint without the need to specify entire path.

Reply:
Thanks. I hadn't thought of using yarn to work out the path. I've updated this to use `yarn run` now.